WordPress/gutenberg
# ToolsPanel: Tighten grid gaps

Follow-up to #43331

## What?

Tighten the row/column gap in ToolsPanel and the Global Styles Typography panel.

## Why?

To match the density of the original Figma mockups.

## How?

This is just a quick fixup. A more sustainable approach will be explored in #43423.

## Testing Instructions

1. `npm run dev`
2. See the Typography tools in both the post editor and Global Styles.

## Screenshots or screencast 

| Before | After |
|--------|-------|
|![Global Styles Typography panel](https://user-images.githubusercontent.com/555336/185608087-99519ba6-862f-45c8-81b5-1c4bb0df4be1.png)|![Global Styles Typography panel with tighter column gap](https://user-images.githubusercontent.com/555336/185608148-23390d35-cd32-4ad6-b3f9-5c64f37831a2.png)|
|![Typography tools](https://user-images.githubusercontent.com/555336/185608275-daca51c6-a69f-49b5-84ef-6c9f131c6dba.png)|![Typography tools with tighter gaps](https://user-images.githubusercontent.com/555336/185608331-302bcd95-a759-4305-ba09-fe762b048f57.png)|
